React/Base/macOS/RCTUIKit.m

```diff
@@ -751,4 +751,93 @@ - (void)setHidden:(BOOL)hidden
 
 @end
 
+// RCTUIImageView
+
+@implementation RCTUIImageView
+
+@synthesize tintColor = _tintColor;
+@synthesize contentMode = _contentMode;
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+  if (self = [super initWithFrame:frame]) {
+    self.layer = [[CALayer alloc] init];
+    self.wantsLayer = YES;
+  }
+  
+  return self;
+}
+
+- (BOOL)clipsToBounds
+{
+  return self.layer.masksToBounds;
+}
+
+- (void)setClipsToBounds:(BOOL)clipsToBounds
+{
+  self.layer.masksToBounds = clipsToBounds;
+}
+
+- (UIViewContentMode)contentMode
+{
+  return _contentMode;
+}
+
+- (void)setContentMode:(UIViewContentMode)contentMode
+{
+  _contentMode = contentMode;
+  switch (contentMode) {
+    case UIViewContentModeScaleAspectFill:
+      self.layer.contentsGravity = kCAGravityResizeAspectFill;
+      break;
+      
+    case UIViewContentModeScaleAspectFit:
+      self.layer.contentsGravity = kCAGravityResizeAspect;
+      break;
+      
+    case UIViewContentModeScaleToFill:
+      self.layer.contentsGravity = kCAGravityResize;
+      break;
+      
+    case UIViewContentModeCenter:
+      self.layer.contentsGravity = kCAGravityCenter;
+      break;
+    
+    default:
+      break;
+  }
+}
+
+- (UIImage *)image
+{
+  return self.layer.contents;
+}
+
+- (void)setImage:(UIImage *)image
+{
+  if (self.layer.contents == image) {
+    return;
+  }
```

Comment:
Also in our style guide is avoiding early returns. Maybe it's worth me adding a link in our docs 🤔

https://microsoft.github.io/objc-guide/MethodsAndImplementations/EarlyReturns.html

Reply:
Shorthand link is aka.ms/objc-guide btw

Reply:
I updated the implementation to match the style guide.